### PR TITLE
add appveyor for CI

### PR DESCRIPTION
**Pull request by rmg**
The environment parts are in place, but the tests and/or test schema themselves might need to be tweaked to account for the environment.

connect to https://github.com/strongloop-internal/scrum-apex/issues/186

### CODE REVIEW

File: test/init.js [hunk 1]
```diff
@@ -21,6 +21,16 @@ try {
   };
 }
 
+if (process.env.APPVEYOR) {
```
Comment (rmg) [inline test/init.js]: Now that we've got this working, should this be changed to use env vars like `MSSQL_HOST`, `MSSQL_PORT`, etc. like the other connectors do and then just set them in `appveyor.yml`?

Comment (ssh24) [inline test/init.js]: 👍. Updated the PR with the changes.

**Comment by ssh24** [comment]
Interesting to see CI failing on windows node 4.

@slnode test please

File: test/tables.sql [hunk 2]
```diff
@@ -7,6 +7,12 @@ drop table session;
 
 go
 
+  CREATE SCHEMA sa;
+
+go
+
+go
```
Comment (0candy) [inline test/tables.sql]: Don't need 2 `go` commands.

Comment (ssh24) [inline test/tables.sql]: Fixed it.

**Pull request merged**